tests-integration/src/tests/test_util.rs

```diff
@@ -110,8 +110,31 @@ pub(crate) async fn check_unordered_output_stream(output: Output, expected: &str
     assert_eq!(pretty_print, expected);
 }
 
-pub fn get_data_dir(path: &str) -> PathBuf {
+pub fn find_testing_resource(path: &str) -> String {
```

Comment:
nitpick: Document the behavior
```suggestion
/// Joins the path to `CARGO_MANIFEST_DIR` and returns a '/' delimited path for object stores.
pub fn find_testing_resource(path: &str) -> String {
```